cmd/clusterctl/clusterdeployer/clusterclient/clusterclient.go

```diff
@@ -110,7 +109,7 @@ func (c *client) removeKubeconfigFile() error {
 func (c *client) EnsureNamespace(namespaceName string) error {
 	clientset, err := clientcmd.NewCoreClientSetForDefaultSearchPath(c.kubeconfigFile, clientcmd.NewConfigOverrides())
 	if err != nil {
-		return fmt.Errorf("error creating core clientset: %v", err)
+		return errors.Errorf("error creating core clientset: %v", err)
```

Comment:
`errors.Wrap` should be used in all places where we are wrapping and there are no extra args.